src/engine/sys/sdl_glimp.cpp

```diff
@@ -1422,6 +1425,82 @@ static void GLimp_CheckGLEW( const glConfiguration &requestedConfiguration )
 	}
 }
 
+static bool IsSdlVideoRestartNeeded()
+{
+	if ( !firstSdlVideoAttempt )
```

Comment:
I don't think this variable needs to exist.

Reply:
I removed it, though I liked the explicitness of it instead of relying on per-workaround restart check.